apps/site/assets/ts/shuttles/__tests__/ShuttlesPageTest.tsx

```diff
@@ -73,3 +91,53 @@ describe("it passes to ShuttlesOverview", () => {
     });
   });
 });
+
+describe("adjusting the selected", () => {
+  let wrapper: ReactWrapper;
+
+  beforeAll(() => {
+    wrapper = mount(<ShuttlesPage route={route} diversions={diversionsData} />);
+  });
+
+  it("direction filters stops on both maps", () => {
+    const stopsLength = diversionsData.stops.length;
+    wrapper.find(ShuttlesMap).forEach(map => {
+      expect(map.prop("stops").length).toBe(stopsLength);
+    });
+
+    act(() => {
+      wrapper
+        .findWhere(n => n.key() === "Alewife")
+        .first()
+        .invoke("onClick")("0");
```

Comment:
It turns out `simulate("click")` works fine in this case, and with my latest commit technically we don't need to update Enzyme here anymore... 🤦‍♀ 

Though [this issue](https://github.com/airbnb/enzyme/issues/2173) suggests `simulate()` will be deprecated someday, and consensus in the Github issues seems to be to avoid using simulate and just call whatever function it's triggering directly... \shrug

Reply:
Good to know! When you say "call the function directly" you mean using `invoke`, or is there another way?

Reply:
Yeah. Probably more relevant for class-based React components, but I think one can define a function on the class component and invoke it by `component.foo()`. Or if the relevant function can be imported, it can be used in the test that way too. Anyway...